deploy/gk-deploy

```diff
@@ -40,20 +43,26 @@ Arguments:
 Options:
   -g, --deploy-gluster
               Deploy GlusterFS pods on the nodes in the topology that contain
-              brick devices. Default is to not deploy GlusterFS, thus assuming
-              that has already been done.
+              brick devices. If the --abort flag is also specified, this flag
+              indicates that all GlusterFS pods and deployments should be
+              deleted as well. Default is to not handle GlusterFS deployment
+              or removal.
 
   -t TEMPLATES, --templates_dir TEMPLATES
               Location of directory containing the heketi templates for
               OpenShift. Default is '${TEMPLATES}'.
 
+  -n NAMESPACE, --namespace NAMESPACE
+              The namespace to use for creating resources.
+              Defaults is '${NAMESPACE}'.
+
```

Comment:
We normally place everything in one namespace, However we can still mention different namespace for a particular resource . Isnt it ? have we tested it ?